src/main/java/fi/fmi/avi/model/immutable/PhenomenonGeometryWithHeightImpl.java

```diff
@@ -10,10 +10,9 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
 import fi.fmi.avi.model.NumericMeasure;
+import fi.fmi.avi.model.PhenomenonGeometryWithHeight;
 import fi.fmi.avi.model.TacOrGeoGeometry;
-import fi.fmi.avi.model.immutable.NumericMeasureImpl;
-import fi.fmi.avi.model.immutable.TacOrGeoGeometryImpl;
-import fi.fmi.avi.model.sigmet.PhenomenonGeometryWithHeight;
+import fi.fmi.avi.model.immutable.PhenomenonGeometryWithHeightImpl_Builder;
```

Comment:
Redundant import. This class could probably be cleaned formatted by FMI IDEA rules.